profile-doc/profile/scribblings/toplevel.scrbl

```diff
@@ -15,7 +15,8 @@ profiler on the @racket[main] submodule of that file (if it exists), or on the
 module itself (if there is no @racket[main] submodule).
 
 To allow control over its behavior, @exec{raco profile} accepts flags that
-correspond to those of @racket[profile-thunk] below.
+correspond to those of @racket[profile-thunk] below, and additional flags that
+control the order of its output.
```

Comment:
I see.
I think it would make sense to add it to the graphviz renderer, too, and just document it as being ignored.